src/main/java/com/autotune/common/datasource/DataSourceInfo.java

```diff
@@ -26,15 +26,17 @@
  * }
  */
 public class DataSourceInfo {
+    private final String name;
     private final String provider;
     private final URL url;
 
-    public DataSourceInfo(String provider, URL url) {
+    public DataSourceInfo(String name, String provider, URL url) {
+        this.name = name;
         this.provider = provider;
         this.url = url;
     }
 
-    public String getProvider() {
```

Comment:
Change this back to using `getProvider` as that is what the variable is now called

Reply:
This should be done at all places in the file